modules/node_modules/@ncigdc/dux/cart.js

```diff
@@ -6,56 +6,165 @@ import { REHYDRATE } from 'redux-persist/constants';
 import _ from 'lodash';
 import UndoIcon from 'react-icons/lib/md/undo';
 
-// Custom
+import { stringify } from 'query-string';
+import { fetchApi } from '@ncigdc/utils/ajax';
 import { notify } from '@ncigdc/dux/notification';
 import { Column } from '@ncigdc/uikit/Flex';
 import { center } from '@ncigdc/theme/mixins';
+import styled from '@ncigdc/theme/styled';
 
 /*----------------------------------------------------------------------------*/
 
 const UPDATE_CART = 'UPDATE_CART';
+const ADD_CART = 'ADD_CART';
+const REMOVE_CART = 'REMOVE_CART';
 
-function toggleInCart(incomingFiles) {
+const NotificationColumn = styled(Column, ({
+  textAlign: 'center',
+  wordBreak: 'break-word'
+}));
+
+const getNotificationComponent = (action, id, notification, dispatch) => ({
+  action: action,
```

Comment:
Can you install `eslint`? I would probably complain about this saying to use object shorthand notation.